tests/test_m2m.py

```diff
@@ -0,0 +1,159 @@
+import unittest
+from m2m import m2m
+
+class MetadataRecordTests(unittest.TestCase):
+
+    def testMetadataRecordSetup(self):
+        # test that the creation of a record creates a MetadataRecord object.
+
+        self.record = m2m.MetadataRecord("mphillips")
```

Comment:
It is probably safer not to attach to `self`. This could lead us to false positives or failures in later tests.